docs/administration/query_queues.md

```diff
@@ -77,12 +137,46 @@ mysql> SHOW PROCESSLIST;
 +------+------+---------------------+-------+---------+---------------------+------+-------+-------------------+-----------+
 ```
 
-- Check the FE audit log file **fe.audit.log**. The field `PendingTimeMs` indicates the time that a query is in a queue, and its unit is milliseconds.
+### FE audit log
+
+You can check the FE audit log file **fe.audit.log**. The field `PendingTimeMs` indicates the time that a query is in a queue, and its unit is milliseconds.
 
-- Check the following FE metrics:
+### FE metrics
 
-| **Metrics**                      | **Description**                                |
-| -------------------------------- | ---------------------------------------------- |
-| starrocks_fe_query_queue_pending | Number of pending queries in the queue.        |
-| starrocks_fe_query_queue_total   | Total number of queries that have been queued. |
-| starrocks_fe_query_queue_timeout | Number of timeout queries in the queue.        |
+The following FE metrics are derived from the statistical data of each FE node.
+
+| Metric                                          | Unit | Type    | Description                                                    |
+| ----------------------------------------------- | ---- | ------- | -------------------------------------------------------------- |
+| starrocks_fe_query_queue_pending                | Count | Instantaneous | The current number of queries in the queue.                  |
+| starrocks_fe_query_queue_total                  | Count | Instantaneous | The total number of queries historically queued (including those currently running). |
+| starrocks_fe_query_queue_timeout                | Count | Instantaneous | The total number of queries that have timed out while in the queue. |
+| starrocks_fe_resource_group_query_queue_total   | Count | Instantaneous | The total number of queries historically queued in this resource group (including those currently running). The `name` label indicates the name of the resource group. This metric is supported from v3.1.4 onwards. |
+| starrocks_fe_resource_group_query_queue_pending | Count | Instantaneous | The number of queries currently in the queue for this resource group. The `name` label indicates the name of the resource group. This metric is supported from v3.1.4 onwards. |
+| starrocks_fe_resource_group_query_queue_timeout | Count | Instantaneous | The number of queries that have timed out while in the queue for this resource group. The `name` label indicates the name of the resource group. This metric is supported from v3.1.4 onwards. |
+
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/98087056/4a5df4cd-c40a-416d-a565-b325b60e879b)

for other two metrics, are they happening in the queue??